src/safe-upgrade-tool/install.md

```diff
@@ -0,0 +1,100 @@
+---
+group: software-update-guide
+title: Install
+ee_only: True
+functional_areas:
+  - Upgrade
+---
+
+The Safe Upgrade Tool ALPHA (SUT) is a command line (CLI) tool that checks a Magento instance against a specific version by analyzing all the non-Magento modules installed on it.
+
+{:.bs-callout-warning}
+At the moment this is an ALPHA version with limited scope, only validating PHP Magento APIs and GraphQL schema.
+
+## Workflow
+
+The following diagram shows the expected workflow when running the SUT:
+
+![SUT Diagram](img/mvp-diagram.png)
```

Comment:
Please replace this image with this one:
![image](https://user-images.githubusercontent.com/10559423/105175070-bf83c900-5b23-11eb-9a23-a139e53211e8.png)

Reply:
added